### DISCUSSION TITLE
Korg N5Ex

### DISCUSSION THREAD

**Discussion by Tadream** [christofmuc/KnobKraft-orm#180] [Q&A]
Any chance of adding the Korg N5ex to the list ?
I am not familiar with programming to try to tackle this myself...

Best regards,

Yves
Belgium

**Comment by christofmuc** [comment]
Hi Yves, thanks for suggesting this! I have never heard of the N5Ex before, interesting!

It seems the information we need is here:
https://www.korg.com/us/support/download/product/1/163/

Errata: https://www.korg.com/us/support/download/manual/1/163/1809/

I am not sure if I understand this manual though, it seems to be yet again different than the other synths I have done so far. 
Maybe we can ask Michal who made the PCG Tools http://pcgtools.mkspace.nl/index.html, I am sure he groked it all. Sadly his software is not open source, but as he said here http://www.korgforums.com/forum/phpBB2/viewtopic.php?t=79386&sid=1b2adab01e065b114c246d6fc31b2c46 that he is not working on the software anymore, maybe he is interested in sharing a bit of his know-how.

I have also not found any example syx files from the N5 do look at, it would help understanding the manual better. Do you have any?

**Comment by christofmuc** [comment]
Ah ok , I found the information. It is in the download zip file, which is a txt file half in Japanese half in English (the English is the translation), and from line 814 following it contains the required sysex messages. We could base this off the user-contributed Korg 03/R adapation or the MS2000 adaptation. That would require a bit of work, but just for the programs it should be fairly straight forward.

@Tadream Would you be able to test an adaptation file with the real device?

**Comment by Tadream** [comment]
Super ! And YES I would be glad to test it ! 

Best regards, 

yves 


Van: "Christof" ***@***.***> 
Aan: "christofmuc/KnobKraft-orm" ***@***.***> 
Cc: "tadream" ***@***.***>, "Mention" ***@***.***> 
Verzonden: Zondag 2 oktober 2022 19:11:40 
Onderwerp: Re: [christofmuc/KnobKraft-orm] Korg N5Ex (Discussion #180) 





Ah ok , I found the information. It is in the download zip file, which is a txt file half in Japanese half in English (the English is the translation), and from line 814 following it contains the required sysex messages. We could base this off the user-contributed Korg 03/R adapation or the MS2000 adaptation. That would require a bit of work, but just for the programs it should be fairly straight forward. 

[ https://github.com/Tadream | @Tadream ] Would you be able to test an adaptation file with the real device?

**Comment by christofmuc** [comment]
Leaving this here, I am not sure at which point this implementation stalled back then. 
[Korg_N5.zip](https://github.com/user-attachments/files/18470590/Korg_N5.zip)
